src/EFCore.PG/Storage/Internal/Mapping/NpgsqlDateTimeTypeMappings.cs

```diff
@@ -110,9 +110,9 @@ protected override RelationalTypeMapping Clone(RelationalTypeMappingParameters p
             => new NpgsqlIntervalTypeMapping(parameters);
 
         protected override string GenerateNonNullSqlLiteral(object value)
-        {
-            var ts = (TimeSpan)value;
-            return FormattableString.Invariant($"INTERVAL '{ts.ToString($@"{(ts < TimeSpan.Zero ? "\\-" : "")}{(ts.Days == 0 ? "" : "d\\ ")}hh\:mm\:ss{(ts.Ticks % 10000000 == 0 ? "" : "\\.FFFFFF")}")}'");
-        }
+            => FormatTimeSpanAsInterval((TimeSpan)value);
+
+        public static string FormatTimeSpanAsInterval(TimeSpan ts)
+            => FormattableString.Invariant($"INTERVAL '{ts.ToString($@"{(ts < TimeSpan.Zero ? "\\-" : "")}{(ts.Days == 0 ? "" : "d\\ ")}hh\:mm\:ss{(ts.Ticks % 10000000 == 0 ? "" : "\\.FFFFFF")}")}'");
```

Comment:
Interpolated string looks like a censored swearing.

Reply:
Do you prefer regex maybe? :rofl:

Reply:
Only imaginary representation of their state machines displayed in a terminal (see [this one](https://twitter.com/thingskatedid/status/1307500428314120193) for example).

Reply:
Lovely :)

I think you'd like the Sqlite documentation: https://sqlite.org/lang_createtable.html